src/qml/qgismobileapp.qml

```diff
@@ -810,22 +810,28 @@ ApplicationWindow {
                   digitizingFeature.geometry.applyRubberband()
                   digitizingFeature.applyGeometry()
                 }
-
                 if( !overlayFeatureFormDrawer.featureForm.featureCreated )
                 {
                     digitizingFeature.resetAttributes();
                     if( overlayFeatureFormDrawer.featureForm.model.constraintsHardValid ){
                       //when the constrainst are fulfilled
-                      digitizingFeature.create()
-                      overlayFeatureFormDrawer.featureForm.featureCreated = true
+                      overlayFeatureFormDrawer.featureForm.featureCreated = digitizingFeature.create()
+
+                      if ( ! overlayFeatureFormDrawer.featureForm.featureCreated ) {
+                        displayToast( qsTr( "Failed to autosave feature!" ) );
```

Comment:
Generally I would display toasts if an action that is made by the user fails (like confirm feature edit etc), and just write log message on actions made in the background by auto save or tracking functionalities.

Reply:
Clear and noted for the future, thanks!